mlc_config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "ignorePatterns": [
+    {
+      "pattern": "^http://localhost"
+    },
+    {
+      "pattern": "^https://banter.pub"
+    }
+  ],
+  "replacementPatterns": [
+    {
+      "pattern": "^/public/images",
+      "replacement": "https://docs.blockstack.org/images"
+    },
+    {
+      "pattern": "^/",
+      "replacement": "https://docs.blockstack.org/"
```

Comment:
Same as other comment

Reply:
replacing relative paths with accessible paths of deployed resources. see the other comment for details :)